src/ui/map.js

```diff
@@ -17,15 +18,18 @@ const bindHandlers = require('./bind_handlers');
 
 const Camera = require('./camera');
 const LngLat = require('../geo/lng_lat');
+/*:: import type LngLatLike from '../geo/lng_lat'; */
```

Comment:
ESlint complains if I try to use the `import` syntax without the flow comment syntax. Some config that needs to change?